snap/snapcraft.yaml

```diff
@@ -1,7 +1,8 @@
 name: doctl
-version: "1.20.0"
+version-script: git tag -l | sort --version-sort | tail -n1
```

Comment:
This output includes the preceding `v`, eg: `v1.20.0` Not sure if that matters, but since we are switching from not having the `v` it could potentially cause issues with version sorting.

Reply:
Nice catch!